kokoro/linux-test/test.sh

```diff
@@ -35,13 +35,7 @@ export CC=/usr/bin/gcc-8
 cd $SRC
 BUILD_SHA=${KOKORO_GITHUB_COMMIT:-$KOKORO_GITHUB_PULL_REQUEST_COMMIT}
 
-function bazel {
-  "${BUILD_ROOT}/bazel/bin/bazel" \
-    --output_base="${TMP}/bazel_out_dbg" \
-    "$@"
-}
-
-function bazel-test {
+function test {
```

Comment:
Can we not call this the same name as a shell builtin which you also use?

Reply:
That is gone now

Reply:
(I was just reverting the changes in test.sh.) Please view all "files changed", not per commit.